yarn.lock

```diff
@@ -13433,6 +13504,20 @@ mjml-core@^4.1.1:
     mjml-parser-xml "^4.1.1"
     mjml-validator "^4.1.1"
 
+mjml-core@^4.2.0:
```

Comment:
When we get into birth certs we should upgrade registry-certs to this version of mjml as well.